front_end/src/components/random_button.tsx

```diff
@@ -31,7 +30,8 @@ const RandomButton: FC = () => {
       disabled={isLoading}
       aria-label="Random Question"
       style={{ background: "transparent" }}
```

Comment:
@lsabor Remove this style and just replace with native `<button>` -- we don't need our specific UI button here